nuget/WindowsAppCenterAnalytics.nuspec

```diff
@@ -21,9 +21,7 @@
   <files>
     <file src="$uwp_dir$/Microsoft.AppCenter.Analytics.dll" target="lib/uap10.0" />
     <file src="$uwp_dir$/Microsoft.AppCenter.Analytics.xml" target="lib/uap10.0" />
-
-    <!-- TODO: Uncomment when it's time to release these. -->
-    <!--file src="$windows_desktop_dir$/Microsoft.AppCenter.Analytics.dll" target="lib/net45" /-->
-    <!--file src="$windows_desktop_dir$/Microsoft.AppCenter.Analytics.xml" target="lib/net45" /-->
+    <file src="$windows_desktop_dir$/Microsoft.AppCenter.Analytics.dll" target="lib/net45" />
```

Comment:
Should update the branch also to avoid showing this diff that was already merged if I am not mistaken.

Reply:
It was merged to this branch, not to develop